pkg/reconciler/workload/resource/resource_reconcile.go

```diff
@@ -72,11 +72,14 @@ func (c *Controller) reconcileResource(ctx context.Context, lclusterName logical
 		return fmt.Errorf("error reconciling resource %s|%s/%s: error getting namespace: %w", lclusterName, obj.GetNamespace(), obj.GetName(), err)
 	}
 
-	annotationPatch, labelPatch := computePlacement(ns, obj)
+	annotationPatch, labelPatch := make(map[string]interface{}), make(map[string]interface{})
```

Comment:
I'd rather have just declared vars here without an instantiation, and instantiate them in the next conditional block.

Reply:
Letting the map be `nil` will have a NPE when you set fields in it, no?

Reply:
Where do we update the maps in this function ? I may have missed it